### PR TITLE
[Feature]: Feedback sharing frontend

### PR DESCRIPTION
**Pull request by OustanDing**
- Banner component
- Notification box on performances page
- Share feedback page
(Features hidden in local branch until API hookup is complete)

Additional changes:
- Refactored `Navigation` container to `Event` container (name is more intuitive and offers easier use for Feedback sharing API hookup)
- `useSessionStorage` hook, essentially the same logic as `useLocalStorage`

image
image
image

### CODE REVIEW

File: pages/feedback.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import Button from '@components/Button'; // Button
+import Table from '@components/Table'; // Table
+import Checkbox from '@components/Checkbox'; // Checkbox
+import Modal from '@components/Modal'; // Modal
+import Event from '@containers/Event'; // Event state
+import styles from '@styles/pages/Feedback.module.scss'; // Component styles
+
+const SAMPLE_DATA = [
+  {
+    schoolName: 'CCI',
```
Comment (thepenguinco) [inline pages/feedback.js:13]: Why not just pull this directly from the schools api?

File: pages/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const modalStates = Object.freeze({
 
 // Page: Events
 export default function Events({ session }) {
-  const { setEvent } = Navigation.useContainer();
+  const [, setEvent] = Event.useContainer();
```
Comment (thepenguinco) [inline pages/index.js:27]: Why ,?

File: pages/performances/[id].js [hunk 3]
```diff
@@ -22,7 +22,7 @@ import { formatPerformance } from '@utils/performances'; // Format performance u
 export default function PerformanceDetails({ session }) {
   const router = useRouter();
   const { id } = router.query;
-  const { event: eventId } = Navigation.useContainer();
+  const [{ id: eventId }] = Event.useContainer();
```
Comment (thepenguinco) [inline pages/performances/[id].js:25]: Some places have event and some places have id: eventId.  Is this intentional?

**Review by thepenguinco — APPROVED** [review]
Small comments, otherwise LGTM.

File: pages/feedback.js (hunk 1 above)
Comment (OustanDing) [reply pages/feedback.js:13]: Going to leave it for the API hookup.

File: pages/index.js (hunk 2 above)
Comment (OustanDing) [reply pages/index.js:27]: We were using `Navigation` just to store the id of the currently selected event. For the feedback sharing, I'd assume that we would have an extra field in the Event object storing any fields related to feedback status, which Feedback Sharing depends on. Also, naming it Navigation was not so clear as to what it was for.

File: pages/performances/[id].js (hunk 3 above)
Comment (OustanDing) [reply pages/performances/[id].js:25]: Yes

**Review by mkanoria — APPROVED** [review]
LGTM! Thanks for refactoring the Navigation container!

**Pull request merged**